scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -501,7 +505,7 @@ final case class ValueSortedReduce[K, V1, V2](
    * After sorting, then reducing, there is no chance
    * to operate in the mappers. Just call take.
    */
-  override def bufferedTake(n: Int) = take(n)
+  override def bufferedTake(n: Int)(implicit q: Quoted) = take(n)
```

Comment:
why do we need a `Quoted` here. Not clear to me. Just to get position information of the caller?